git-link.el

```diff
@@ -211,9 +211,10 @@ As an example, \"gitlab\" will match with both \"gitlab.com\" and
   (car (git-link--exec "--no-pager" "log" "-n1" "--pretty=format:%H")))
 
 (defun git-link--commit ()
-  (if (git-link--using-git-timemachine)
-      (car git-timemachine-revision)
-    (git-link--last-commit)))
+  (cond
+   ((git-link--using-git-timemachine) (car git-timemachine-revision))
+   ((git-link--using-magit-blame) (oref (magit-blame-chunk-at (point)) :orig-rev))
```

Comment:
I don't see any documentation for `magit-blame-chunk-at`. Maybe it's not part of public interface? I see docs for `magit-blame-chunk-get`.  I'm using version `20171022.9`, maybe this is why?

Reply:
> I'm using version 20171022.9, maybe this is why?

Probably. I'm running magit from source, but `(magit-version)` returns `v2.90.1-701-gc9a754fa`. 

The corresponding release is this one: https://github.com/magit/magit/releases/tag/v2.90.1

It will be a bit difficult to support older magit versions - so this feature will probably be gated on `magit-blame-chunk-at` being available. Do you think this is a blocker for this PR?

> Maybe it's not part of public interface?

Yes, I didn't check that. When I update this PR (shouldn't take long) I'll take this into account. Thanks for the reminder!